src/components/info-tag/InfoTag.tsx

```diff
@@ -2,16 +2,25 @@ import {StyleProp, ViewStyle} from 'react-native';
 import {StyleSheet} from '@atb/theme';
 import React from 'react';
 import {InfoChip} from '../info-chip';
+import {InfoTagTexts, useTranslation} from '@atb/translations';
 
 type Props = {
-  text: string;
+  text: 'beta' | 'new' | string; // Restricting text to either 'beta', 'new' or any other string
```

Comment:
Not a fan of mixing texts and "modes" this way. Maybe look at some other way of doing it.

Maybe the `InfoTag` component can have predefined modes like `new` and `beta`, and the `InfoChip` can still just take in `string`?